.github/workflows/ci.yml

```diff
@@ -29,46 +29,46 @@ jobs:
     strategy:
       matrix:
         eslint: [7]
-        node: [14.x]
+        node: [14]
         os: [ubuntu-latest]
         include:
           # On other platforms
           - eslint: 7
-            node: 14.x
+            node: 14
             os: windows-latest
           - eslint: 7
-            node: 14.x
+            node: 14
             os: macos-latest
           # On old Node.js versions
           - eslint: 7
-            node: 12.x
+            node: 12
             os: ubuntu-latest
           - eslint: 7
-            node: 10.x
+            node: 10
             os: ubuntu-latest
           # On old ESLint versions
           - eslint: 6
-            node: 14.x
+            node: 14
             os: ubuntu-latest
           - eslint: 5
-            node: 14.x
+            node: 14
             os: ubuntu-latest
           - eslint: 4
-            node: 14.x
+            node: 14
             os: ubuntu-latest
           # On the minimum supported ESLint/Node.js version
           - eslint: 4
-            node: 8.10.0
+            node: 8.10
             os: ubuntu-latest
 
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
       - name: Install Node.js ${{ matrix.node }}
         uses: actions/setup-node@v1
         with:
-          node-version: ${{ matrix.node }}
+          node: ${{ matrix.node }}
```

Comment:
The original `node-version` is the correct parameter name.

Reply:
The checkout step just uses `node`, that's why I changed it here too

Reply:
For example, https://github.com/mysticatea/eslint-plugin-es/runs/666749068?check_suite_focus=true#step:3:5, we can see the `actions/setup-node` ignores the `node` parameter.